examples/bidder/bidder.hpp

```diff
@@ -70,7 +70,7 @@ namespace vanilla {
             bid.id = boost::uuids::to_string(bidid); // TODO check documentation 
             // Is it the same as response.bidid?
             bid.impid = imp.id;
-            bid.price = ad->max_bid_micros / 1000000.0; // Not micros?
+            bid.price = ad->auth_bid_micros ? *ad->auth_bid_micros / 1000000.0 : ad->max_bid_micros / 1000000.0 ; // Not micros?
```

Comment:
shall we wrap the price converting code to an inline utility function with readable name?